.kodiak.toml

```diff
@@ -1,8 +1,8 @@
 version = 1
 
 [merge]
-automerge_label = "🚀 merge it!"
-blocking_labels = ["wip", "do-not-merge"]
+automerge_label = ["🚀 merge it!", "Merge It"]
```

Comment:
Me again about this, Rich 😅 
If the label is being used in the PR when will the PR be merged? When it has 1 approval?

Reply:
PRs now require 2 approvals. Once all checks pass, and there are two approvals it should auto-merge.